odxtools/diagcodedtype.py

```diff
@@ -70,7 +70,7 @@ def is_highlow_byte_order(self) -> bool:
         return self.is_highlow_byte_order_raw in [None, True]
 
     @staticmethod
-    def _extract_internal_value(
+    def _extract_atomic_value(
```

Comment:
Why?

Reply:
I wanted a different name: there are/were so many methods with "internal" in their name that I get confused quite  easily. (Also, I think that the new value better conveys the fact that this method returns an `AtomicOdxType` object...)